playwright/pages/draftOrdersPage.ts

```diff
@@ -60,6 +60,14 @@ export class DraftOrdersPage extends BasePage {
   }
 
   async clickAddShippingCarrierButton() {
+    await this.page.waitForLoadState("networkidle");
+
+    await this.addShippingCarrierLink.waitFor({
+      state: "visible",
+      timeout: 10000,
```

Comment:
Do not add hardcoded timeout, rather we should rethink the test if it stucks somwhere.